### PR TITLE
HTML and CSS code

### PR DESCRIPTION
**Pull request by jainpriya123**
Hey Please review PR!

### CODE REVIEW

File: NotesApp/NotesApp.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <link rel="stylesheet" href="NotesApp.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+<body>
+    <div class="head">
+        <div class="container">
+            <div class="heading">
+                <div class="text"><i class="fa-sharp fa-solid fa-pen-to-square"></i><span><b>Notes</b></span></div>
+            </div>
+
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.html]: remove this extra line

File: NotesApp/NotesApp.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                <div class="text"><i class="fa-sharp fa-solid fa-pen-to-square"></i><span><b>Notes</b></span></div>
+            </div>
+    
+            <div class="button">
+                <button class="btn" type="button"><i class="fa-solid fa-pencil"></i><span>Create Notes</span></button>
+            </div>
+        </div>
+    </div>
+    <script type="text/javascript" src="NotesApp.js">
+
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.html]: remove this line

File: NotesApp/NotesApp.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    gap: 5px;
+    align-items: center;
+}
+
+.button {
+    display: flex;
+    width: 150px;
+}
+
+.pen {
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.css]: remove this unused class

File: NotesApp/NotesApp.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+<body>
+    <div class="head">
+        <div class="container">
+            <div class="heading">
+                <div class="text"><i class="fa-sharp fa-solid fa-pen-to-square"></i><span><b>Notes</b></span></div>
+            </div>
+    
+            <div class="button">
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.html]: don't class name same as any html tag

File: NotesApp/NotesApp.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
+* {
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.css:1]: give more meaningful and explanatory names to classes

review everywhere for this

like head does not makes any sense

File: NotesApp/NotesApp.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+                <button class="btn" type="button"><i class="fa-solid fa-pencil"></i><span>Create Notes</span></button>
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.html]: do we need to specify type="button" inside button tag

File: NotesApp/NotesApp.html [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="NotesApp.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+<body>
+    <div class="head">
+        <div class="container">
+            <div class="heading">
+                <div class="text"><i class="fa-sharp fa-solid fa-pen-to-square"></i><span><b>Notes</b></span></div>
```
Comment (AkashdeepGarg) [inline NotesApp/NotesApp.html]: if <b> is used , then no need to embed it inside <span> tag

**Review by AkashdeepGarg — CHANGES_REQUESTED** [review]

File: NotesApp/NotesApp.css (hunk 5 above)
Comment (jainpriya123) [reply NotesApp/NotesApp.css:1]: I have removed head

Comment (jainpriya123) [reply NotesApp/NotesApp.css:1]: I have changed the name

File: NotesApp/NotesApp.css (hunk 3 above)
Comment (jainpriya123) [reply NotesApp/NotesApp.css]: I have removed.

**Pull request merged**